api/server/creds.go

```diff
@@ -11,61 +12,155 @@ import (
 func (vd *volAPI) credsEnumerate(w http.ResponseWriter, r *http.Request) {
 	method := "credsEnumerate"
 
-	d, err := vd.getVolDriver(r)
+	ctx, err := vd.annotateContext(r)
+
 	if err != nil {
-		notFound(w, r)
+		vd.sendError(vd.name, method, w, err.Error(), http.StatusInternalServerError)
 		return
 	}
 
-	creds, err := d.CredsEnumerate()
+	// Get gRPC connection
+	conn, err := vd.getConn()
 	if err != nil {
 		vd.sendError(vd.name, method, w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	json.NewEncoder(w).Encode(creds)
+
+	client := api.NewOpenStorageCredentialsClient(conn)
+	resp, err := client.Enumerate(ctx, &api.SdkCredentialEnumerateRequest{})
+
```

Comment:
nit: Remove empty line

Reply:
Thats weird, in mu IDE there is no blank line.